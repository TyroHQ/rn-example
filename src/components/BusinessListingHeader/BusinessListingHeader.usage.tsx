import React from "react";
import { boolean, text, number } from "@storybook/addon-knobs";
import { action } from "@storybook/addon-actions";
import { BusinessListingHeader } from "./BusinessListingHeader";

export const basic = () => <BusinessListingHeader />;
