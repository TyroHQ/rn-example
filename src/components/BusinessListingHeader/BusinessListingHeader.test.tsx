import { shallow } from "enzyme";
import * as usage from "./BusinessListingHeader.usage";

describe("<BusinessListingHeader />", () => {
  for (const example in usage) {
    it(`renders well in ${example} mode`, () => {
      const x = (usage as any)[example]();
      const wrapper = shallow(x);
      expect(wrapper).toMatchSnapshot();
    });
  }
});
