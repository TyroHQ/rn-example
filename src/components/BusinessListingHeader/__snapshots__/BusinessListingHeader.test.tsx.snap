// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<BusinessListingHeader /> renders well in basic mode 1`] = `
<Placeholder
  orientation="row"
>
  Business listing container
  <Placeholder
    orientation="row"
  >
    Left White container
    <CornerBanner />
    <Placeholder
      orientation="row"
    >
      Business logo
    </Placeholder>
  </Placeholder>
  <Placeholder
    orientation="row"
  >
    Right green container
    <Placeholder
      orientation="row"
    >
      Content frame
      <Placeholder
        orientation="row"
      >
        Left aligned items
        <Placeholder
          orientation="row"
        >
          Business name H4
        </Placeholder>
        <Placeholder
          orientation="row"
        >
          Business type H5
        </Placeholder>
        <Placeholder
          orientation="row"
        >
          City (Distance in km) H6
        </Placeholder>
      </Placeholder>
      <Placeholder
        orientation="row"
      >
        Offer text
      </Placeholder>
    </Placeholder>
    <Placeholder
      orientation="row"
    >
      Down icon
    </Placeholder>
  </Placeholder>
</Placeholder>
`;
